src/datasets/providers/devcycle.ts

```diff
@@ -36,7 +36,7 @@ export const DevCycle: Provider = {
       category: ['Client'],
     },
     {
-      technology: 'React',
+      technology: ['React', 'JavaScript'],
```

Comment:
Won't this create duplicate providers in the ecosystem page? Perhaps that's okay but it would artificially inflate the numbers.

Reply:
with the latest changes, it doesn't create duplicate entries if a provider has a React provider and a base web provider: 
<img width="1588" alt="image" src="https://github.com/user-attachments/assets/69870120-f655-4c62-bd93-edace346547f" />